### DISCUSSION TITLE
How do I elevate the user viewpoint relative to the landscape?

### DISCUSSION THREAD

**Discussion by Donathan-west** [Stellarium/stellarium#3195] [Q&A] [answered]
Hi, I'm using a 3d landscape, which I generated myself.  But relative to the landscape, I am seeing the buildings from about 3 feet off the ground, which changes what is visible in the sky. 

How do I elevate the user viewpoint relative to the landscape?

**Comment by gzotti** [comment, marked as answer]
The first paragraph of section 15.2 of the User Guide tells you basic navigation. Don't forget to display coordinates, which is described in paragraph 2. Section 15.4.3 includes a description of the start_Eye parameter which defaults to 1.65m, not 3 feet.

**Comment by JChase67** [comment]
Find the landscape folder for the specific landscape you are using:  >(Windows file explorer)  >C:\  >Program files (x86)  >Stellarium  >landscapes  >(your landscape)...
Use a text editor (like notepad) to edit the .ini file inside that folder.  Change the 'maptex_top' parameter to the value you want.  Restart Stellarium.
Recommend you make a backup copy of this file before you make changes so you can return to original setup if needed.
